package.json

```diff
@@ -20,11 +20,17 @@
   "bin": {
     "gherkin-testcafe": "./main.js"
   },
+  "main": "src/createTestcafeGherkin.js",
   "dependencies": {
-    "cucumber-expressions": "^5.0.7",
+    "async-exit-hook": "^2.0.1",
+    "endpoint-utils": "^1.0.2",
     "gherkin": "^5.0.0",
     "glob": "^7.1.2",
-    "testcafe": "^0.18.5",
+    "stack-chain": "^2.0.0",
+    "testcafe": "^0.19.2",
     "yargs": "^10.0.3"
+  },
+  "peerDependencies": {
+    "cucumber": "^4.2.1"
```

Comment:
The least benchmark for this package should be the ability to run 

```
node main.js --specs ./examples/**/*.feature --steps ./examples/**/*.js
```

Just after checking out the source and installing all dependencies.

In order for this to work, `cucumber` needs to be added as a devDependency as well.